react/components/SearchResult.js

```diff
@@ -138,6 +146,8 @@ class SearchResult extends Component {
       showLoadingAsOverlay,
     } = this.state
 
+    const PAGINATION_TYPES = ['show-more', 'infinite-scroll']
```

Comment:
put this outside the component so it will be instantiated just once

Reply:
and change it to

`const PAGINATION_TYPE_SHOW_MORE = 'show-more'`

since the other value is not being used